mycore-base/src/main/java/org/mycore/frontend/jersey/MCRJWTUtil.java

```diff
@@ -55,18 +57,31 @@
     private static Algorithm SHARED_SECRET;
 
     public static JWTCreator.Builder getJWTBuilder(MCRUserInformation userInformation) {
+        return getJWTBuilder(userInformation, null);
+    }
+
+    public static JWTCreator.Builder getJWTBuilder(MCRUserInformation userInformation, String[] userAttributes) {
```

Comment:
better use vararg `String... userAttributes` here, without any userattributes one can simply call getJWTBuilder(userInformation) and omit null as a second parameter

Reply:
nice, I didn't know that, then I can save the adapter.